Bottom: ce7954e341082ec5c8249e494d1f33a77dff6375
Top:    65f42312bd58f218afc8ed030c8c7356927f8bfc
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 17:00:20 +0100

Refresh of hid-gtk-free-list-returned-fro

---

diff --git a/src/hid/gtk/ghid-main-menu.c b/src/hid/gtk/ghid-main-menu.c
index 6b58485..d816498 100644
--- a/src/hid/gtk/ghid-main-menu.c
+++ b/src/hid/gtk/ghid-main-menu.c
@@ -519,7 +519,7 @@ void
 ghid_main_menu_install_layer_selector (GHidMainMenu *mm,
                                        GHidLayerSelector *ls)
 {
-  GList *children, iter;;
+  GList *children, *iter;
 
   /* @layerview */
   if (mm->layer_view_shell)
@@ -563,7 +563,7 @@ void
 ghid_main_menu_install_route_style_selector (GHidMainMenu *mm,
                                              GHidRouteStyleSelector *rss)
 {
-  GList *children;
+  GList *children, *iter;
   /* @routestyles */
   if (mm->route_style_shell)
     {
